plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -111,6 +118,11 @@ public static IcebergFileFormat getFileFormat(Map<String, Object> tablePropertie
         return (IcebergFileFormat) tableProperties.get(FILE_FORMAT_PROPERTY);
     }
 
+    public static IcebergMetadataCompressionCodec getMetadataCompressionCodec(Map<String, Object> tableProperties)
+    {
+        return (IcebergMetadataCompressionCodec) tableProperties.get(METADATA_COMPRESSION_CODEC);
```

Comment:
to be on the safe side - maybe a previous version of Iceberg didn't return this value, should we maybe use `getOrDefault` `IcebergMetadataCompressionCodec.NONE`